setup.sh

```diff
@@ -26,22 +28,16 @@ apt-get install -qy \
     vim \
     virtualenvwrapper
 
-# Add ubuntu to the passwordless sudo file.
-echo "ubuntu ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/juju-users
+# Add the JUJU_USER to the passwordless sudo file.
+echo "${JUJU_USER} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/juju-users
+
 JUJU=`which juju-1 > /dev/null && echo juju-1 || echo juju`
 # Get the version of Juju installed.
 JUJU_VERSION=`${JUJU} version`
-# The home directory for the ubuntu user.
-HOME=/home/ubuntu
-# Set some environment variables used by Juju.
-RC=${HOME}/.bashrc
-cat << EOF > $RC
-# The directory to look for charms.
-export JUJU_REPOSITORY=${HOME}
-# The path to Juju configuration files.
-export JUJU_HOME=${HOME}/.juju
```

Comment:
oh, i see in master that these aren't set in the .bashrc either.. do `ENV` doohickeys persist when the image is run?  if so, i see why these .bashrc additions aren't needed.